ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tensor.py

```diff
@@ -70,10 +73,266 @@ def _check_query(query):
     )
 
 
+def _helper_init_tensorarray(backend_fw, l_kwargs, fn=None):
+    id_write, kwargs = l_kwargs
+    with BackendHandler.update_backend(backend_fw) as ivy_backend:
+        local_importer = ivy_backend.utils.dynamic_import
+        function_module = local_importer.import_module(
```

Comment:
I think naming this `tf_frontend`, and then naming the tensor array variables `ta` and `ta_gt` would be more comprehensible. But that's subjective I guess.

Reply:
Yes I guess `tf_frontend` is more clear and using `gt` rather than `frontend` is more in line with convention being followed in the testing pipeline, will make changes